sql/sql_analyze_stmt.h

```diff
@@ -49,22 +49,22 @@ class Exec_time_tracker
 {
 protected:
   ulonglong count;
-  ulonglong cycles;
+  ulonglong ns;
```

Comment:
I would suggest a more verbose name, which would give who reads this a clear explanation of what the variable is. 
```suggestion
  ulonglong time_interval_in_ns;
```